src/libopal.c

```diff
@@ -2392,6 +2392,42 @@ short print_asm_code(asm_cmd_e cmd_list[], FILE *dest_fp)
   /// Copy NASM footer file to dest_fp
 
   /// Create strings and their lengths
+  for (int i = 0; i < strs_len; i++)
+    {
+      fprintf (dest_fp, "msg%d: DB \"", i);
+      /// Read each string character
+      for (int j = 0; j < strlen(strs[i])+1; j++)
+        {
+           ///print ASCII values for newlines
+           if (strs[i][j] == 10)
```

Comment:
You are again checking if the stream has a newline .. we want to check if user wants to print a newline in the resulting binary.

```
if (strs[i][j] == '\\' && strs[i][j + 1] == 'n')
  {
    fprintf (dest_fp, "\", 13, 10, \"");
    j = j + 1;
  }
```

Reply:
We are going to pretend it is my second covid dose that caused this confusion, and not the fact that I misunderstood this the whole time ;)